### PR TITLE
Fixes #22679 - job template for reboot to use shutdown -r +1

### PR DESCRIPTION
**Pull request by pmoravec**
Signed-off-by: Pavel Moravec <pmoravec@redhat.com>

### CODE REVIEW

File: job_templates/power_action_-_ssh_default.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@ template_inputs:
 echo <%= input('action') %> host && sleep 3
 <%= case input('action')
       when 'restart'
-        'reboot'
+        'shutdown -r +1'
```
Comment (ekohl) [inline job_templates/power_action_-_ssh_default.erb:19]: How does +1 compare to now?

**Comment by ekohl** [comment]
Could you also change https://github.com/theforeman/community-templates/blob/fdae1ff37493e8ce794720c841ac296a99a2ed52/job_templates/power_action_-_ansible_default.erb#L25

File: job_templates/power_action_-_ssh_default.erb (hunk 1 above)
Comment (pmoravec) [reply job_templates/power_action_-_ssh_default.erb:19]: "shutdown -r now" is basically the same like "reboot" - either command can trigger shutting down network *before* the command terminates, causing the "ssh" command from remote server will not receive any return value and will be waiting for it frever.

**Comment by pmoravec** [comment]
updated the ansible job template accordingly.

**Review by dLobatog — APPROVED** [review]

**Review by iNecas — APPROVED** [review]

**Comment by iNecas** [comment]
Tested on centos7 and rhel5, works well on both.

**Review by dLobatog — COMMENTED** [review]
Thanks @pmoravec !

**Pull request merged**